JS/viewController.js

```diff
@@ -1,40 +1,32 @@
-/// ViewController is abstract class that represents one application screen
 class ViewController {
 
     constructor() {
         this.root = $('#root');
         this.navigationController;
     }
 
-    /// Presents actual app screen. Call present and it will draw HTML
-    /// it will setup properties, event listeners and finally call viewDidLoad
     present() {
         this.renderHtml();
         this.setupProperties();
         this.setupEventListeners();
         this.viewDidLoad();
     }
 
-    /// Renders HTML
-    /// @param html - html you are going to draw, in extended view controller
-    ///               you must call super.renderHtml(html)
+    // Renders HTML
```

Comment:
Ehm, tomuto trochu nerozumiem, to co si tu porobil? ... mam pocit ze si tu vymazal nejake moje subory, je tu zmazana moja dokumentacia a namiesto nej napisana nejaka renders HTML a podobne, to znamena ze ked sa toto mergne do mastra tak mi vsetky moje veci prepise.. to preco?